src/validators/v1/rollouts.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * name : validators/v1/rollouts.js
+ * author : Priyanka Pradeep
+ * Date : 27-Nov-2024
+ * Description : Validations of rollout controller
+ */
+const common = require('@constants/common')
+const filterRequestBody = require('../common')
+const { rollouts } = require('@constants/blacklistConfig')
+module.exports = {
+	update: (req) => {
+		req.body = filterRequestBody(req.body, rollouts.update)
+
+		if (req.method != common.REQUEST_METHOD_DELETE) {
+			req.checkBody('title').trim().notEmpty().withMessage('title is required')
+		}
+
+		req.checkParams('id')
+			.trim()
+			.optional({ checkFalsy: true })
+			.notEmpty()
+			.withMessage('id param is empty')
+			.isNumeric()
+			.withMessage('id param is invalid, must be an integer')
+			.isInt({ min: 1, max: 2147483647 })
+			.withMessage('Id is not valid')
+
+		req.checkParams('start_date').trim().notEmpty().withMessage('start_date param is empty')
```

Comment:
@priyanka-TL  start_date is passed in the body right?  not as a param. Please recheck this file.